.prettierrc

```diff
@@ -0,0 +1,6 @@
+{
+  "printWidth": 80,
+  "tabWidth": 2,
+  "singleQuote": true,
+  "trailingComma": "all"
```

Comment:
Can we change it? I don't like much having things like:

```
{
  "foo": "bar",
}
```

Reply:
While changing something in an object with trailing comma, the diff is more readable. But it's up to you to change it as the owner of this repo.😉

Reply:
Also the default recommended behaviour on Prettier.

Reply:
It is actually breaking https://github.com/brownbaglunch/java-webhook project.
I did consider that I could easily read a structure like:

```js
var data = {
  "speakers": [
    {
      "since": "2012-11-05"
    }
  ]
}
```

And transform it easily as a JSON document with some simple String manipulation:

```json
{
  "speakers": [
    {
      "since": "2012-11-05"
    }
  ]
}
```

But with this:

```js
var data = {
  speakers: [
    {
      since: '2012-11-05',
    }
  ]
}
```

This is now producing things like:

```json
{
  speakers: [
    {
      since: '2012-11-05',
    }
  ]
}
```

Which is no more valid as a JSON document.

I'd really like if we can change the behavior. CC @linsolas

Reply:
Trailing comma is part of ECMAScript 5.1 specification *([11.1.4](https://www.ecma-international.org/ecma-262/5.1/#sec-11.1.4) & [11.1.5](https://www.ecma-international.org/ecma-262/5.1/#sec-11.1.5))*, and resolve a lot of issue regarding object initialization. *(diffs, reordering, appending, concatenating…)*

By the way you should always use a real api method (like `JSON.stringify()`) for converting `js` literal objects to `JSON` *(serialization)*, and never just *transform* `js` to `json` by string manipulations as it has a lot of chance to be invalid. (js to json by string manipulation just works in a few edge cases and is quite a bad practice).

Reply:
Thanks @mabhub. I'll give it a try in https://github.com/brownbaglunch/java-webhook/blob/master/src/main/java/fr/brownbaglunch/webhook/data/BblDataReader.java#L72-L75 and see how it goes.